slimta/util/encoders.py

```diff
@@ -0,0 +1,91 @@
+# -*- coding: utf-8 -*-
+
+import six
+
+__all__ = ['encode_base64']
+
+
+def utf8only_decode(bytestr):
+    """ Decode bytes assuming it's utf-8, raising no exception
+
+    Part of the "be liberal n what you accept" policy
+
+    :param unistr: a string where no non-utf8 char is supposed to occur
+    :type unistr: str
+    :rtype bytes:
+    """
+    return bytestr.decode('utf-8', 'replace')
+
+
+printable_decode = utf8only_decode
+
+
+def utf8only_encode(unistr):
+    """ Encode to ascii
+
+    Silently replaces invalid chars with "�"
+
+    :param unistr: a string where no non-utf8 char is supposed to occur
```

Comment:
I think this param description should be more like `a string that may contain non-ascii characters`